src/middleware/transactionBody.js

```diff
@@ -0,0 +1,26 @@
+import mongodb from 'mongodb'
+
+export const retrieveTransactionBody = (db, fileId, callback) => {
+  if (!db) {
+      const err = new Error(`Transaction body retrieval failed. Database handle: ${db} is invalid`)
+      return callback(err, null)
+  }
+
+  if (!fileId) {
+    const err = new Error(`Transaction body retrieval failed: Transaction id: ${fileId}`)
```

Comment:
This error isnt very descriptive as to why it failed. The description also makes reference to Transaction which it shouldnt be, as this function could be used for any GridFS document retrieval